docs/pelion.md

```diff
@@ -0,0 +1,34 @@
+# Pelion
+
+## Requirements
+
+kube-router documentation [General Requirements](https://www.kube-router.io/docs/user-guide/#requirements)
+
+Make sure ipset is installed
+```
+apt update && apt install -y ipset
+```
+
+## Build binary without docker
+```
+BUILD_IN_DOCKER=false make kube-router
```

Comment:
Should this be the default method to build kube-router? Shouldn't we also add the method to build binary with docker?

Reply:
Added more build information and with/without docker alternatives.